vault/core.go

```diff
@@ -2868,17 +2869,22 @@ func (c *Core) SetLogLevel(level log.Level) {
 	}
 }
 
-func (c *Core) SetLogLevelByName(name string, level log.Level) error {
+// SetLogLevelByName sets the logging level of named logger to level provided
```

Comment:
This is a great godoc. Thanks for adding those.

Reply:
Of course, thanks for the feedback :)